Sources/Sentry/SentryOptions.m

```diff
@@ -14,60 +14,86 @@ @implementation SentryOptions
     ];
 }
 
+- (instancetype)init
+{
+    if (self = [super init]) {
+        self.enabled = @NO;
+
+        // TODO: Remove this side effect
+        SentrySDK.logLevel = kSentryLogLevelError;
```

Comment:
H: why are we accessing the static class inside of the Options init? I see the TODO, but It'd be nice to have a comment about the reason for it instead only "side effect"

Reply:
This code was already there and I just moved it from `validateOptions`. I didn't think too much about it, that's why I added a todo to investigate it.

Reply:
I started getting rid of this but it's not a 5 min task. This static log is accessed everywhere and moving to the approach we took on Android will take time.
Will raise a task for this

Reply:
wfm